src/json/mapping.cr

```diff
@@ -80,7 +80,7 @@ module JSON
       {% end %}
 
       {% if value[:getter] == nil ? true : value[:getter] %}
-        def {{key.id}}
+        def {{key.id}} : {{value[:type]}} {{ (value[:nilable] ? "?" : "").id }}
```

Comment:
`?` should come directly after the type (`Bool?` not `Bool ?`)

Reply:
This is exactly as it is in the setter above. Should both be changed?

Reply:
It parses with whitespace, but it would be better to remove it.